### DISCUSSION TITLE
Why can't i just upload a doc for proof of student standing for the education pack?

### DISCUSSION THREAD

**Discussion by coffeebeantraining** [community/community#53333] [GitHub Education]
I am trying to get verified for the student pack but the only option is to take a picture on my computer and upload.
Is there another way to upload proof of academic enrollment?  Taking a picture on my computer camera is not an option because my computer camera is low quality and github continually denies, whereas other platforms such as AWS and Azure just use your email as verification.
I have a pdf version of my transcript, but the camera shows too much blur.

![image](https://user-images.githubusercontent.com/59173960/233452301-af65df48-5e93-4a47-81d2-6e583b2ca43d.png)

**Comment by samus-aran** [comment]
Hi @coffeebeantraining 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

It would be preferable that a student/teacher is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students/teachers submit a live photo of their evidence rather than an upload. We understand that students/teachers may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy. Ensure you have as much of the document in the image (mug shots aren't as important 😄 )
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
4. If you have trouble take a photo via your laptop you can submit your application through your mobile.

Let us know here if you need further assistance

**Reply by yacc143** [reply]
Problem is, I actually have access to a government signed PDF that proves my status. But I cannot upload it.

But as our universities are very helpful, they only call the document "STUDY RECORD SHEET AND STUDY PERIO CONFIRMATION for the Wintersemester 2023/2024". The email your edubot mailed me contained some mentioning that my proof does not contain any date of reference. You can even validate the document against government servers that it's the genuine thing and a government signed document. (Austrian universities are mostly state universities, and thus dual headed and can act as government authorities)

> Ok I take that back, the "verification process for my university" is [send us the scanned document]l(https://www.jku.at/en/degree-programs/students/administrative-signature/) and we'll tell you if we issued it, ah, digitalization in the year 2023 can be a joke, still relying on that nobody would dare to fake a stamp on an official document.

Sure it does. It references semesters. And that's how Austrian universities deal with that. Guess your "AI" script is to stupid to understand these?

The only official (and don't take it wrong, that one is trivial to fake) thing that references a date are student IDs that are plastic cards where the validity is erased by a special printer and reprinted. And as a remote student I don't have these yet, I haven't participated yet in any meetings at the uni this semester, the only mandatory ones are the exams in January for the winter semester.

And now I'm stuck with a verification process that insists that I cannot even submit it, where can I get help?

Andreas

**Reply by tsouchlarakis** [reply]
Was this ever resolved?

**Reply by sri-ae21b058** [reply]
In my case, I have a dated ID card but the date and my photo are on opposite sides which doesn't help in approval

**Reply by alioshr** [reply]
Still not resolved. This is unbelievable. This should be very easy to implement. I will need to take a picture from my phone pointing to a PDF on my computer screen

**Comment by patriciafraga** [comment]
Em acréscimo: qual a camada superior de segurança que a foto de um pdf assinado (digitalmente) irá agregar? Não está fazendo muito sentido.

**Reply by yacc143** [reply]
No they are not making sense, they are following a near definition of how to identify a student that might make sense in some countries, and much less in others.

**Comment by yacc143** [comment]
Yeah, they don't really care much about winning "young" minds.

Not that it matters much, I'm going with the selfhost/OSS route anyway,
it's in the current world much better than working with big tech tools.

Am Do., 7. Nov. 2024 um 22:24 Uhr schrieb malik-gaci <
***@***.***>:

**Comment by marcosmonaco** [comment]
Currently having this problem, Im from Argentina and the best proof I can give on my School is the Official Transcript, but that is a PDF, which I cant upload, when I merge the two pages of said document to a single JPEG picture, it then says that the image is edited and I need to use my camera to show the PDF... Honestly, I think they just want this to be as hard as possible in order to make more people pay for the benefits.

**Reply by sri-ae21b058** [reply]
I have a similar problem with my ID too with my photo and the valid date in my id card being on opposite sides

**Comment by yacc143** [comment]
Shrug, I went with selfhosting forgejo, I just need to figure out how to
put let's encrypt reverse proxying on the docker stack, then I can have a
second "publicly" visible instance, problem solved.

This internet of the huge silos of huge corporate giants isn't that cool
anyways.

Am Di., 14. Jan. 2025 um 08:31 Uhr schrieb Marcos Monaco <
***@***.***>:

**Comment by JoseFSMelo** [comment]
Até hoje ainda sem solução desse caso...
Não consigo fazer captura pelo celular, e não consigo upar o documento pelo computador, bizarro.

**Comment by DrHMAlanazi** [comment]
Why  they don't enable automatic verification through the university portal or allow the upload of official PDF files?
STILL UNSOLVED

**Comment by kadirzbk** [comment]
lütfen izin verin

**Comment by nickksoares** [comment]
It shocks me that they are trying so hard to stop you from even trying, nowadays few schools in my country give physical IDs everything is online, so when I send a photo of me besides my phone with the enrollment letter and application it does not recognize anything. Thank god the Big company owned by the First and Second Greek letter holding is giving 15mo of cloud and AI.
